momentum_scanner_intraday.py

```diff
@@ -1,11 +1,16 @@
 import time
-
+import json
 import pandas as pd
 
 import cache_db
 import gecko
 import metrics
 
+network_map_cg = {'Ethereum':'ethereum','BSC':'binance-smart-chain','Polygon':'polygon-pos','Arbitrum':'arbitrum-one'}
```

Comment:
Constants in python should be `SCREAMING_SNAKE_CASE`. Format this PR, sort the entries.

A comment on what these constants are used for would be helpful.